example/main.go

```diff
@@ -282,28 +289,28 @@ func archiveItem(client *onepassword.Client, vaultID string, itemID string) {
 
 func generateItemSharing(client *onepassword.Client, vaultID string, itemID string) string {
 	// [developer-docs.sdk.go.item-share-get-item]-start
-	item, err := client.Items.Get(context.Background(), vaultID, itemID)
+	item, err := client.Items().Get(context.Background(), vaultID, itemID)
 	if err != nil {
 		panic(err)
 	}
 	// [developer-docs.sdk.go.item-share-get-item]-end
 
 	// [developer-docs.sdk.go.item-share-get-account-policy]-start
-	accountPolicy, err := client.Items.Shares.GetAccountPolicy(context.Background(), item.VaultID, item.ID)
+	accountPolicy, err := client.Items().Shares().GetAccountPolicy(context.Background(), item.VaultID, item.ID)
```

Comment:
I must say that this actually looks more aesthetic to me 🚀 . When you also add the testing capabilities we enable for developers who can now mock all these interfaces, this v0 breaking change seems really worth it to me.